src/main/java/frc/robot/commands/ClimberEnable2022CMD.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.commands;
+
+import frc.robot.logging.RobotLogManager;
+import frc.robot.subsystems.Climber2022;
+
+import org.apache.logging.log4j.Logger;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ClimberEnable2022CMD extends CommandBase {
+    private final Climber2022 climber;
+
+    private static final Logger LOGGER = RobotLogManager.getMainLogger(Climber2022.class.getName());
```

Comment:
the logger is using the wrong class
```suggestion
    private static final Logger LOGGER = RobotLogManager.getMainLogger(ClimberEnable2022CMD.class.getName());
```